common/services/app/civic-uk.ts

```diff
@@ -15,12 +15,20 @@ export const getAnalyticsConsentState = (
   const consentCookie = cookies.CookieControl;
 
   // Ensures this returns true for regular users
-  // that don't have the "Cookie works" toggle on/have defined the consent cookie
-  if (isCookiesWorkToggleOn && consentCookie !== undefined) {
-    const civicUKCookie: CivicUKCookie = JSON.parse(
-      decodeURIComponent(consentCookie)
-    );
-    return civicUKCookie.optionalCookies?.analytics === 'accepted';
+  // that don't have the "Cookie works" toggle on
+  if (isCookiesWorkToggleOn) {
+    // If the feature flag is ON and consent has been defined,
+    // return its value
+    if (consentCookie !== undefined) {
+      const civicUKCookie: CivicUKCookie = JSON.parse(
+        decodeURIComponent(consentCookie)
+      );
+
+      return civicUKCookie.optionalCookies?.analytics === 'accepted';
+    } else {
+      // If the feature flag is ON but consent has yet to be defined
+      return false;
+    }
   } else {
     return true;
```

Comment:
In the future we'll need the default no consent state to be false, worth adding a comment to be double sure we do that?

Reply:
When we remove the toggle and therefore the first condition, and only keep the second condition, with a change to the comments, that's what it'll do 👍 

```
if (consentCookie !== undefined) {
    const civicUKCookie: CivicUKCookie = JSON.parse(
      decodeURIComponent(consentCookie)
    );

    return civicUKCookie.optionalCookies?.analytics === 'accepted';
  } else {
    // If consent has yet to be defined, return false by default
    return false;
  }
}
```

We'll for sure test that a lot, it's why we'll requisition e2e for a bit, with all the toggle stuff out of the way 🤞